src/floras/components/transition_system.py

```diff
@@ -16,12 +16,13 @@ class TransitionSystemInput():
         labels: Labels of each system state.
         init: Initial state of the system.
     """
-    def __init__(self,states,transitions,labels, init):
+    def __init__(self,states,transitions,labels, init, custom_map = None):
```

Comment:
```suggestion
    def __init__(self,states,transitions,labels, init, custom_map=None):
```
[Recommendation of PEP 8](https://peps.python.org/pep-0008/#other-recommendations) is no space around `=` before default value.

Reply:
I am linting right now, the next PR will include all the linted code (or I will update this one if I get it done quickly).